packages/core/src/parsers.ts

```diff
@@ -1,21 +1,13 @@
-import { CCSSParser, CCSSPipe } from '@/types'
+import { CCSSInput, CCSSParser, CCSSPipe } from '@/types'
 
-export const toCSSRule = (cssProp, objectProp) => {
-    return (input, prop, options) => {
-        return options.outputTransformer.toCSSRule(cssProp, objectProp, input, prop, options)
-    }
-}
+export const toCSSRule = (cssProp: string, objectProp: string): CCSSParser => (input, prop, options) =>
```

Comment:
I don't have short version on purpose, it has to be rewritten every time for debugging.